sdk/servicebus/service-bus/test/invalidParameters.spec.ts

```diff
@@ -551,224 +586,220 @@ describe("Invalid parameters in SessionReceiver #RunInBrowser", function(): void
-    let caughtError: Error | undefined;
-    try {
-      await receiver.receiveDeferredMessage(undefined as any);
-    } catch (error) {
-      caughtError = error;
-    }
-    should.equal(caughtError && caughtError.name, "TypeError");
-    should.equal(caughtError && caughtError.message, `Missing parameter "sequenceNumber"`);
-  });
-
-  it("ReceiveDeferredMessages: Wrong type sequenceNumbers in Receiver", async function(): Promise<
-    void
-  > {
-    let caughtError: Error | undefined;
-    try {
-      await receiver.receiveDeferredMessages(["somestring"] as any);
-    } catch (error) {
-      caughtError = error;
-    }
-    should.equal(caughtError && caughtError.name, "TypeError");
-    should.equal(
-      caughtError && caughtError.message,
-      `The parameter "sequenceNumbers" should be an array of type "Long"`
-    );
-  });
-
-  it("ReceiveDeferredMessages: Missing sequenceNumbers in Receiver", async function(): Promise<
-    void
-  > {
-    let caughtError: Error | undefined;
-    try {
-      await receiver.receiveDeferredMessages(undefined as any);
-    } catch (error) {
-      caughtError = error;
-    }
-    should.equal(caughtError && caughtError.name, "TypeError");
-    should.equal(caughtError && caughtError.message, `Missing parameter "sequenceNumbers"`);
-  });
-
-  it("RenewMessageLock: Missing lockTokenOrMessage in Receiver", async function(): Promise<void> {
-    let caughtError: Error | undefined;
-    try {
-      await (<InternalReceiver>receiver).renewMessageLock(undefined as any);
-    } catch (error) {
-      caughtError = error;
-    }
-    should.equal(caughtError && caughtError.name, "TypeError");
-    should.equal(caughtError && caughtError.message, `Missing parameter "lockTokenOrMessage"`);
-  });
-
-  it("RenewMessageLock: Invalid string lockToken in Receiver", async function(): Promise<void> {
-    let caughtError: Error | undefined;
-    try {
-      await (<InternalReceiver>receiver).renewMessageLock("string-which-is-not-uuid");
-    } catch (error) {
-      caughtError = error;
-    }
-    should.equal(
-      caughtError && caughtError.message,
-      `Not a valid UUID string: string-which-is-not-uuid`
-    );
-  });
-
-  it("RenewMessageLock: Invalid message lockToken in Receiver", async function(): Promise<void> {
-    let caughtError: Error | undefined;
-    try {
-      const [receivedMsg] = await receiver.receiveMessages(1);
-      if (!receivedMsg) {
-        throw new Error("Message not received to renew lock on.");
-      }
-      (<any>receivedMsg).lockToken = "string-which-is-not-uuid";
-      await (<InternalReceiver>receiver).renewMessageLock(receivedMsg);
-    } catch (error) {
-      caughtError = error;
-    }
-    should.equal(
-      caughtError && caughtError.message,
-      `Not a valid UUID string: string-which-is-not-uuid`
-    );
-  });
-});
+// These tests are exactly same as the previous describe block - session vs non-session
```

Comment:
```suggestion
// These tests are exactly same as the previous describe block - session vs non-session.
// Since the current 2-client API version doesn't differentiate between session vs session (w.r.t the methods), there is no need for duplication.
// This is subject to change when the top-level client is implemented.
```